client/db/src/bench.rs

```diff
@@ -97,7 +102,10 @@ impl<B: BlockT> BenchmarkingState<B> {
 	fn reopen(&self) -> Result<(), String> {
 		*self.state.borrow_mut() = None;
 		self.db.set(None);
-		let db_config = DatabaseConfig::with_columns(1);
+		let mut db_config = DatabaseConfig::with_columns(1);
+		if let Some(size) = &self.cache_size_mb {
+			db_config.memory_budget.insert(0, *size);
```

Comment:
This is not a real benchmark

Cache should be cleared, not disabled

Reply:
It is not disabled, but rather enabled here. `0` is column index. `wipe` reopens the database so it gets cleared.